components/ledger/ledger-utxo-flow/src/main/kotlin/net/corda/ledger/utxo/flow/impl/persistence/UtxoLedgerPersistenceServiceImpl.kt

```diff
@@ -128,8 +128,9 @@ class UtxoLedgerPersistenceServiceImpl @Activate constructor(
         }
     }
 
-    private fun SignedTransactionContainer.toSignedTransaction(): UtxoSignedTransaction {
-        return utxoSignedTransactionFactory.create(wireTransaction, signatures)
+    private fun SignedTransactionContainer.toSignedTransaction(utxoLedgerPersistenceService: UtxoLedgerPersistenceService)
+    : UtxoSignedTransaction {
+        return utxoSignedTransactionFactory.create(wireTransaction, signatures, utxoLedgerPersistenceService)
```

Comment:
```suggestion
        return utxoSignedTransactionFactory.create(wireTransaction, signatures, this)
```

Seems a little odd to take `utxoLedgerPersistenceService` as an argument, when it is `this` object and this function is private? Suggested approach results in being able to revert the change to line 62 as well.

Reply:
Updated here: https://github.com/corda/corda-runtime-os/pull/2819/files#diff-6ca8a2c0e6178df692e3bc6b4d24858f329a12a609ffa359752d20bc07971377